backend/internal/server/apis/schedule/games/games.go

```diff
@@ -95,7 +118,10 @@ func optimizeSchedule(games []models.Game) {
 	}
 }
 
-func generateGames(leagues []models.League, numberOfGamesPerTeam int) models.Season {
+func generateGames(leagues []models.League, numberOfGamesPerTeam int) (models.Season, error) {
+	if len(leagues) == 0 {
+		return models.Season{}, errors.New("no leagues to games games for")
```

Comment:
I like putting in all these checks

Reply:
Thank you I thought they where good for stablity